man/zeek-client.1

```diff
@@ -0,0 +1,165 @@
+.TH ZEEK\-CLIENT "1" "2023\-09\-08" "zeek\-client" "Generated Python Manual"
+.SH NAME
+zeek\-client
+.SH SYNOPSIS
+.B zeek\-client
+[-h] [-c FILE] [--controller HOST:PORT] [--set SECTION.KEY=VAL] [--quiet | --verbose] [--version] {deploy,deploy-config,get-config,get-id-value,get-instances,get-nodes,monitor,restart,stage-config,show-settings,test-timeout} ...
+.SH DESCRIPTION
+A command\-line client for Zeek's Management Framework.
+
+Use this client to push cluster configurations to a cluster controller, retrieve
+running state from the system, restart nodes, and more.
+
+For details about Zeek's Management Framework, please consult the documentation at
+https://docs.zeek.org/en/master/frameworks/management.html.
+
+.SH OPTIONS
+.TP
+\fB\-c\fR \fI\,FILE\/\fR, \fB\-\-configfile\fR \fI\,FILE\/\fR
+Path to zeek\-client config file. (Default: @ZEEK_CLIENT_CONFIG_FILE@)
+
+.TP
+\fB\-\-controller\fR \fI\,HOST:PORT\/\fR
+Address and port of the controller, either of which may be omitted (default: 127.0.0.1:2149)
+
+.TP
+\fB\-\-set\fR \fI\,SECTION.KEY=VAL\/\fR
+Adjust a configuration setting. Can use repeatedly. See show\-settings.
+
+.TP
+\fB\-\-quiet\fR, \fB\-q\fR
+Suppress informational output to stderr.
+
+.TP
+\fB\-\-verbose\fR, \fB\-v\fR
+Increase informational output to stderr. Repeat for more output (e.g. \-vvv).
+
+.TP
+\fB\-\-version\fR
+Show version number and exit.
+
+.SH
+COMMANDS
+.SS \fBzeek\-client deploy\fR
+Deploy a staged cluster configuration.
+
+usage: build.py deploy [\-h]
+.SS \fBzeek\-client deploy\-config\fR
+Upload a cluster configuration and deploy it.
+
+usage: build.py deploy\-config [\-h] FILE
+
+arguments:
+.RS 7
+.TP
+\fBFILE\fR
+Cluster configuration file, "\-" for stdin
+.RE
+
+.SS \fBzeek\-client get\-config\fR
+Retrieve staged or deployed cluster configuration.
+
+usage: build.py get\-config [\-h] [\-\-filename FILE] [\-\-as\-json] [\-\-deployed | \-\-staged]
+
+options:
+.RS 7
+.TP
+\fB\-\-filename\fR \fI\,FILE\/\fR, \fB\-f\fR \fI\,FILE\/\fR
+Output file for the configuration, default stdout
+
+.TP
+\fB\-\-as\-json\fR
+Report in JSON instead of INI\-style config file
+
+.TP
+\fB\-\-deployed\fR
+Return deployed configuration
+
+.TP
+\fB\-\-staged\fR
+Return staged configuration (default)
+.RE
+
+.SS \fBzeek\-client get\-id\-value\fR
+Show the value of a given identifier in Zeek cluster nodes.
+
+usage: build.py get\-id\-value [\-h] IDENTIFIER [NODES ...]
+
+arguments:
+.RS 7
+.TP
+\fBIDENTIFIER\fR
+Name of the Zeek script identifier to retrieve.
+
+.TP
+\fBNODES\fR
+Name(s) of Zeek cluster nodes to query. When omitted, queries all nodes.
+.RE
+
+.SS \fBzeek\-client get\-instances\fR
+Show instances connected to the controller.
+
+usage: build.py get\-instances [\-h]
```

Comment:
It looks like setting `parser.prog` in `man/build.py` is insufficient, not exactly sure how to fix that.

```python
    # The parser currently has this script as program name, so replace:
    parser.prog = "zeek-client"
```


Here and below.

Reply:
Oh nice — I missed that! I'm fixing it by changing the name in `sys.argv`, prior to the parser generation.